src/applications/caregivers/components/NeedHelpFooter/index.jsx

```diff
@@ -1,55 +1,42 @@
 import React from 'react';
 import { links } from 'applications/caregivers/definitions/content';
 
-const NeedHelpFooter = () => {
-  const sectionClassNames =
-    'vads-u-margin-y--2p5 vads-u-display--flex vads-u-flex-direction--column';
-  return (
-    <footer className="need-help-footer row vads-u-padding-x--1p5">
+const NeedHelpFooter = () => (
+  <footer className="need-help-footer row vads-u-padding-x--1p5">
+    <div style={{ maxWidth: '600px' }}>
       <h5>Need Help?</h5>
       <hr />
-      <span>Need help filling out this form? Call our toll-free number: </span>
+      <p>
+        You can call the VA Caregiver Support Line at{' '}
+        <a href={links.caregiverHelp.phoneLink}>
+          {links.caregiverHelp.phoneLabel}
+        </a>
+        . We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m. ET.
+      </p>
 
-      <section className={sectionClassNames}>
-        <a href={links.VAHelpLine.label}>{links.VAHelpLine.link}</a>
-        <span>
-          TTY:{' '}
-          <a href={links.VAHelpLine.linkTTL}>{links.VAHelpLine.labelTTL}</a>
-        </span>
-        <span>Monday through Friday, 8:00a.m. — 8:00p.m. ET.</span>
-      </section>
+      <p>
+        You can also call our main VA information line at{' '}
+        <a href={links.VAHelpLine.label}>{links.VAHelpLine.link}</a>, or contact
+        your local Caregiver Support Coordinator.
+      </p>
 
-      <section className={sectionClassNames}>
-        <span>A Caregiver Support Coordinator locator is available at: </span>
+      <span>
         <a
-          href={links.caregiverHelpPage.link}
           target="_blank"
           rel="noopener noreferrer"
+          href="https://www.caregiver.va.gov/support/New_CSC_Page.asp"
         >
-          {links.caregiverHelpPage.link}
-        </a>
-      </section>
-
-      <section className={sectionClassNames}>
-        <span>
-          You can contact the National Caregiver Support Line by calling:
```

Comment:
Looks like there should be a space between "calling:" and the phone number. Something like `{' '}` should work.

Reply:
I think this is fixed? going to run through and remove these spaces and use CSS classes.